inference-engine/samples/benchmark_app/CMakeLists.txt

```diff
@@ -5,9 +5,23 @@
 file (GLOB SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
 file (GLOB HDR ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
 
+find_package(HddlUnite QUIET)
+if(HddlUnite_FOUND)
+    # test kmb_plugin directory
+    if(EXISTS ${KMBPLUGIN_HOME}/include/)
+        set(HDDL2_PARAMS_DIR ${KMBPLUGIN_HOME}/include/)
+        add_definitions(-DUSE_REMOTE_MEM)
```

Comment:
I think you can use `target_compile_definitions`

Reply:
Thanks, I've updated.